tutorials/files/underwater_vehicles/buoyant_lrauv.sdf

```diff
@@ -271,16 +271,10 @@
         </plugin>
 
         <!-- Uncomment to add hydrodynamics -->
-        <!-- <plugin
+        <plugin
```

Comment:
should this be uncommented? if so, we can remove the `<!-- Uncomment to add hydrodynamics -->` line above

Reply:
Resolved in https://github.com/gazebosim/gz-sim/commit/6a235ee810b5e6f857309caf8a0e7af2f3ae23dc